MetaMask/metamask-extension
# Include snap version in pill

## Explanation

Moves the snap version to the pill component for snap install.

## Screenshots/Screencaps



### Before

![image](https://user-images.githubusercontent.com/1561200/171429951-9353b858-d822-4e80-a0cb-3c7d7d82bd99.png)

### After

![image](https://user-images.githubusercontent.com/1561200/171430048-6e098f92-e3a6-4bdb-8f84-eee242486c2e.png)

## Manual Testing Steps

1. Go to https://metamask.github.io/test-snaps/0.3.0/
2. Connect and install a snap
3. See that the snap version is included in the pill
